gax/test/unit/grpc-fallback.ts

```diff
@@ -355,28 +341,44 @@ describe('grpc-fallback', () => {
           JSON.stringify(err.statusDetails),
           JSON.stringify(expectedError.details),
         );
+        done();
```

Comment:
I'm not sure how quickly it was failing before, but right now even with the done in place, if the assertion error fails, it times out instead of failing fast